app/views/conservation_records/treatment_report_pdf.html.erb

```diff
@@ -92,15 +92,15 @@ table {
 
   <h2>In-House Repairs</h2>
   <ul>
```

Comment:
In the PDF, I noticed that we're using an unordered list and passing numbered values to it - which appears to result in a list with bullets and numbers:
<img width="361" alt="Screen Shot 2020-03-17 at 8 18 26 AM" src="https://user-images.githubusercontent.com/1069588/76855859-658de500-6828-11ea-8676-15da3630fb09.png">

Maybe we should pass values without numbers and use `<ol>` tag? This is minor I suppose.

Reply:
I think that's very doable.  I'm not sure why that didn't occur to me the first time around.